apps/frontend/src/app/Header.tsx

```diff
@@ -334,7 +336,7 @@ function MobileHeader({
                   component={RouterLink}
                   to={to}
                   selected={currentTab === value}
-                  disabled={currentTab === value}
+                  disabled={currentTab === value && !inTeam}
```

Comment:
Why not allow navigating back all the time? We allow it in desktop view, I don't really see a reason to disable it

Reply:
It acts as more of an indicator to which page the user is on. we could probably style the buttons differently to indicate, but this is the fastest solution

Reply:
Could make it blue like desktop

Reply:
could, if you feel strongly about it, raise an issue. this PR is mainly address the bug issue #1925.

Reply:
its like 3 line change, and it'll have to undo this